frappe/public/js/frappe/form/quick_entry.js

```diff
@@ -70,6 +72,8 @@ frappe.ui.form.QuickEntryForm = class QuickEntryForm {
 				df.fieldtype !== "Tab Break"
 			);
 		});
+
+		this.mandatory = this.docfields; // backward compatibility
```

Comment:
Some existing QuickEntry classes might overwrite the `this.mandatory` reference, instead of updating it in-place.

https://github.com/frappe/erpnext/blob/38568a3e150496a4292ebb1386b45c0795d825e4/erpnext/public/js/utils/contact_address_quick_entry.js#L12

```js
this.mandatory = this.mandatory.concat(this.get_variant_fields());
```

So I believe the getter/setter approach in https://github.com/frappe/frappe/pull/27344 might be best for the time being

Reply:
Thanks! pushed another fix.